mindsdb/integrations/handlers/db2_handler/db2_handler.py

```diff
@@ -0,0 +1,301 @@
+                        
+                    )
+                )
+            else:
+                response = Response(RESPONSE_TYPE.OK)
+            
+        except Exception as e:
+            log.error(f'Error running while getting table {e} on ')
+            response = Response(
+                RESPONSE_TYPE.ERROR,
+                error_message=str(e)
+            )
+
+
+
+        return response
+
+    
+    def get_columns(self, table_name: str) -> StatusResponse:
+        """ Returns a list of entity columns
+        Args:
+            table_name (str): name of one of tables returned by self.get_tables()
+        Returns:
+            HandlerResponse: shoud have same columns as information_schema.columns
+                (https://dev.mysql.com/doc/refman/8.0/en/information-schema-columns-table.html)
+                Column 'COLUMN_NAME' is mandatory, other is optional. Hightly
+                recomended to define also 'DATA_TYPE': it should be one of
+                python data types (by default it str).
+        """
+        
+        self.connect()
+
+
+        result=self.connection.columns(table_name=table_name)
+        try:
+            if result:
+                response = Response(
+                    RESPONSE_TYPE.TABLE,
+                    data_frame=pd.DataFrame(
+                        [result[i]['COLUMN_NAME'] for i in range(len(result)) ],
+                        columns=['COLUMN_NAME']
+                        
+                    )
+                )
+            else:
+                response = Response(RESPONSE_TYPE.OK)
+            
+        except Exception as e:
+            log.error(f'Error running while getting table {e} on ')
+            response = Response(
+                RESPONSE_TYPE.ERROR,
+                error_message=str(e)
+            )
+
+
+
+        return response
+
+        
+
+
+
+
+
+connection_args = OrderedDict(
+    host={
+        'type': ARG_TYPE.STR,
+        'description': 'The host name or IP address of the DB2 server/database.'
+    },
+    database={
+        'type': ARG_TYPE.STR,
+        'description': """
+            The database name to use when connecting with the DB2 server.
+        """
+    },
+    user={
+        'type': ARG_TYPE.STR,
+        'description': 'The user name used to authenticate with the DB2 server.'
+    },
+    password={
+        'type': ARG_TYPE.STR,
+        'description': 'The password to authenticate the user with the DB2 server.'
+    },
+    port={
+        'type': ARG_TYPE.INT,
+        'description': 'Specify port to connect DB2 through TCP/IP'
+    },
+    schemaName={
+        'type': ARG_TYPE.STR,
+        'description': 'Specify the schema name '
+    },
+
+)
+
+connection_args_example = OrderedDict(
+    host='127.0.0.1',
+    port='25000',
+    password='1234',
+    user='db2admin',
+    schemaName="db2admin",
+    database="BOOKS",
+)
+
+# 💙💙💛💛💜💜🤍🤎🤍🤍🤎🤎🤎❤️❤️❤️❤️❤️🧡🧡🧡💜💜❤️❤️❤️🤍🤎🤍🤍❤️🧡💙💙
```

Comment:
Please just remove this, and I will merge. I've tried CREATE PREDICTOR  too and everything works

Reply:
removed it.

But I think it looks nice ;-}